ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Services/Abstract/IProjectService.cs

```diff
@@ -12,5 +12,9 @@ public interface IProjectService
         Task<ProjectDTO> GetByIdAsync(Guid id);
 
         Task<ProjectDTO> CreateAsync(CreateProjectDTO project);
+
+        Task<List<string>> GetNamesAsync();
+
+        Task<string[]> GetProjectActivitiesAsync(Guid id);
```

Comment:
let's transform Activities from array to List in model, dto, and all methods, so that your code will be in one style. Now GetNames returns list, Activities array, not consistent

Reply:
GetProjectActivitiesAsync you can rename to GetActivitiesAsync, it's obvious that Project because this method in ProjectService